packages/deployment/lib/kes.js

```diff
@@ -179,9 +196,261 @@ class UpdatedKes extends Kes {
       dest
     ).then(() => {
       this.Lambda.messageAdapterZipFileHash = new this.Lambda(this.config).getHash(src);
-    }).then(() => super.compileCF());
+      return this.superCompileCF();
+    });
+  }
+
+  /**
+   * Modified version of Kes superclass compileCF method
+   *
+   * Compiles a CloudFormation template in Yaml format.
+   *
+   * Reads the configuration yaml from `.kes/config.yml`.
+   *
+   * Writes the template to `.kes/cloudformation.yml`.
+   *
+   * Uses `.kes/cloudformation.template.yml` as the base template
+   * for generating the final CF template.
+   *
+   * @returns {Promise} returns the promise of an AWS response object
+   */
+  async superCompileCF() {
```

Comment:
Ignoring the obvious code dupe issues addressed in the follow-on tickets, I'm deeply unsatisfied with any name I've come up with for this function.  Suggestions?